setup.cfg

```diff
@@ -45,3 +45,4 @@ per-file-ignores =
     */ep407/rev0*/a/main.py:E402
     */ep418/rev0*/t.py:F821
     */ep430/rev0*/t.py:F821
+    */ep433/rev0*/t.py:E999
```

Comment:
This one is a gross, but I guess that it's needed 😂

Reply:
wait what's E999 O_o

Reply:
oh invalid syntax, right